tokio/src/io/util/copy_bidirectional.rs

```diff
@@ -0,0 +1,116 @@
+cfg_io_util! {
```

Comment:
This is already around the `mod copy_bidirectional;` statement, so it is not necessary here. Additionally, it breaks rustfmt.